homework/03LessonThree.js

```diff
@@ -89,22 +93,22 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = [ 1, 2, 3, 4, 5 ];
 
 // DO NOT MODIFY
-var nameArray = ['bob', 'fred', 'susan'];
+var nameArray = ['bob', 'george', 'susan'];
```

Comment:
Do not modify threeByThreeArrray or nameArray directly.

You need to use bracket or dot notation to access and assign values to these arrays.